governance/veANGLE/boost.md

```diff
@@ -0,0 +1,111 @@
+
+## Boost calculations
+
+Basically, the contract will consider that an address holding veANGLE is providing more liquidity than it really is. This will increase their share of the pool, increasing the rewards received. As usual, LPs holding no veANGLE are considered as providing 100% of their liquidity. If an address owns enough veANGLE, the contract can consider that it brings up to 250% their original liquidity, or a x2.5 boost compared to the original 100%.
+
+Then, this x2.5 multiplier in liquidity provided will translate into a boost on rewards depending on the total liquidity provided in the pool and how it is considered by the boost calculator.
+
+{% hint style="info" %}
+Note that as the quantity of veANGLE owned by LPs on a pool increase, the amount of rewards received by non-veANGLE holders decreases, as they represent a smaller share of the liquidity.
+{% endhint %}
+
+### Details
+
+#### Understanding the formula
+
+Let:
+
+$$
+x_{lp} = \texttt{my liquidit in this pool} \\
+tot_{lp} = \texttt{total liquidity in this pool} \\
+x_{veANGLE} = \texttt{my veANGLE balance} \\
+s_{veANGLE} = \texttt{total veANGLE supply}
+$$
+
+The complete formula to compute the liquidity provided and the boost applied is:
+
+$$
+\begin{equation}
+\min(x_{lp} + 1.5\times tot_{lp}\times \frac{x_{veANGLE}}{s_{veANGLE}}, 2.5 \times x_{lp})
+\end{equation}
+$$
+
+(similar to Curve’s one)
+
+As we can see, an address owning 0 veANGLE will get rewards as usual. To get the max possible boost on rewards, the address needs to get 100% of its provided liquidity taken into account by the protocol. This is equivalent to have the term on the left greater or equal than the value of the liquidity provided:
+
+$$
+\begin{equation}
+x_{lp} + 1.5\times tot_{lp}\times \frac{x_{veANGLE}}{s_{veANGLE}} \geq 2.5 \times x_{lp}
+\end{equation}
+$$
+
+$$
+\begin{equation}
+\cancel{1.5}\times tot_{lp}\times \frac{x_{veANGLE}}{s_{veANGLE}} \geq \cancel{1.5} \times x_{lp}
+\end{equation}
+$$
+
+$$
+\begin{equation}
+\frac{x_{veANGLE}}{s_{veANGLE}} \geq \frac{x_{lp}}{tot_{lp}}
+\end{equation}
+$$
+
+$$
+\begin{equation}
+\texttt{share of veANGLE supply} \geq  \texttt{share of liquidity in the pool}
+\end{equation}
+$$
+
+**As we can see, an address having a higher share of the total veANGLE supply than its share of liquidity provided in the pool for which the boost is being computed will be considered as having x2.5 more liquidity than an address with no veANGLE.**
+
+From these formulas, we conclude that to get the maximum rewards boost, an address needs to hold at least similar share of the veANGLE total supply than its share of the pool.
+
+#### Examples
+
+Now that we understand how veANGLE holders get a boost on capital compared to non-holders, let’s look at how this translates into an **actual boost on rewards received.**
```

Comment:
I think we should avoid "we" in docs. The style should be a bit more formal than articles